src/System.Text.Json/tests/Utf8JsonReaderTests.MultiSegment.cs

```diff
@@ -893,7 +872,7 @@ public static void SkipSingleLineCommentMultiSpanTest(string lineSeparator)
                 var json = new Utf8JsonReader(sequence.Slice(0, i), isFinalBlock: false, state);
                 VerifyReadLoop(ref json, null);
 
-                json = new Utf8JsonReader(sequence.Slice(state.BytesConsumed), isFinalBlock: true, state);
+                json = new Utf8JsonReader(sequence.Slice(json.BytesConsumed), isFinalBlock: true, json.CurrentState);
```

Comment:
Fyi @WinCPP, these tests were not doing what was intended.

`state.BytesConsumed` is always 0 unless you refresh it by getting the state from the `Utf8JsonReader` again (via `CurrentState`)